package.json

```diff
@@ -1,1375 +1,1385 @@
+		"vscode:prepublish:ext": "webpack --mode production",
+		"vscode:prepublish:view": "webpack --mode production --config src/components/logs/webpack.config.js",
+		"lint": "eslint -c .eslintrc.js --ext .ts ./src",
+		"compile": "npm run compile:ext && npm run compile:view",
+		"compile:ext": "webpack --mode none",
+		"compile:view": "webpack --mode none --config src/components/logs/webpack.config.js",
+		"watch": "webpack --mode development --watch --info-verbosity verbose",
+		"test-compile": "tsc -p ./",
+		"test": "npm run test-compile && node ./out/test/runTest.js"
+	},
+	"extensionDependencies": [
+		"redhat.vscode-yaml"
+	],
+	"dependencies": {
+		"@kubernetes/client-node": "^0.14.0",
+		"ajv": "^6.9.1",
+		"await-notify": "^1.0.1",
+		"clipboardy": "^1.2.3",
+		"compare-versions": "^3.1.0",
+		"debug": "^3.1.0",
+		"docker-file-parser": "^1.0.3",
+		"dockerfile-parse": "^0.2.0",
+		"download": "^7.1.0",
+		"fuzzysearch": "^1.0.3",
+		"got": "^11.8.2",
+		"graceful-fs": "^4.1.11",
+		"js-yaml": "^3.13.1",
+		"lodash": "^4.17.21",
+		"mixin-deep": "^1.3.2",
+		"mkdirp": "^0.5.1",
+		"moment": "^2.29.1",
+		"natives": "^1.1.3",
+		"node-yaml-parser": "^0.0.9",
+		"opn": "^5.2.0",
+		"pluralize": "^4.0.0",
+		"portfinder": "^1.0.13",
+		"rxjs": "^6.5.4",
+		"semver": "^5.5.1",
+		"shelljs": "^0.7.7",
+		"spawn-rx": "^3.0.0",
+		"sshpk": "^1.13.2",
+		"tar": "^4.4.1",
+		"tmp": "^0.0.31",
+		"unzipper": "^0.10.5",
+		"url-parse": "^1.5.1",
+		"uuid": "^3.1.0",
+		"vscode-debugadapter": "1.27.0",
+		"vscode-debugprotocol": "1.27.0",
+		"vscode-extension-telemetry": "^0.1.1",
+		"vscode-uri": "^1.0.1",
+		"yaml-ast-parser": "^0.0.40",
+		"yamljs": "^0.3.0"
+	},
+	"devDependencies": {
+		"@bendera/vscode-webview-elements": "^0.2.0",
+		"@types/clipboardy": "^1.1.0",
+		"@types/js-yaml": "^3.12.0",
+		"@types/lodash": "^4.14.113",
+		"@types/mkdirp": "^0.5.2",
+		"@types/mocha": "^8.0.4",
+		"@types/node": "^10.2.0",
+		"@types/opn": "^5.1.0",
+		"@types/pluralize": "^0.0.29",
+		"@types/request": "^2.48.1",
+		"@types/semver": "^5.5.0",
+		"@types/shelljs": "^0.7.8",
+		"@types/sinon": "^7.0.0",
+		"@types/tar": "^4.0.0",
+		"@types/tmp": "^0.0.33",
+		"@types/unzipper": "^0.10.0",
+		"@types/uuid": "^3.4.4",
+		"@types/vscode": "^1.52.0",
+		"@types/websocket": "^0.0.40",
+		"@types/yamljs": "^0.2.30",
+		"@typescript-eslint/eslint-plugin": "^4.28.1",
+		"@typescript-eslint/eslint-plugin-tslint": "4.28.1",
+		"@typescript-eslint/parser": "^4.28.1",
+		"electron": "^13.1.4",
+		"eslint": "^7.29.0",
+		"file-loader": "^6.2.0",
+		"gulp": "^4.0.2",
+		"gulp-tslint": "^8.1.2",
+		"html-webpack-plugin": "^4.2.0",
+		"mocha": "^8.1.3",
+		"sinon": "^6.3.5",
+		"ts-loader": "^6.0.4",
+		"tslint": "^5.18.0",
+		"typescript": "^3.5.2",
+		"vscode-test": "^1.4.1",
+		"webpack": "^4.35.2",
+		"webpack-cli": "^3.3.5"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools"
+	},
+	"__metadata": {
```

Comment:
Not sure where this came from - is it intentional?

Reply:
Not sure either. It wasn't intentional